Lab2.ipynb

```diff
@@ -0,0 +1,805 @@
+      "text/plain": [
+       "<Figure size 1800x720 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "k = np.linspace(0,20,21)\n",
+    "poisson = stats.poisson.pmf(k,mu=CR_Background)\n",
+    "\n",
+    "summed_poisson = convolve(poisson,n=5)\n",
+    "\n",
+    "x = np.linspace(0,100,101)\n",
+    "poisson_true = stats.poisson.pmf(x,mu=10)\n",
+    "\n",
+    "plt.rcParams[\"figure.figsize\"] = (25,10)\n",
+    "fig, ax = plt.subplots(1, 2)\n",
+    "plt.tick_params(labelsize = 10)\n",
+    "\n",
+    "ax[0].step(x,summed_poisson, where='mid',label = '5 Summed Poisson Distributions, $\\lambda$ = 2')\n",
+    "ax[0].step(x,poisson_true, where='mid',label = 'Poisson Distribution, $\\lambda$ = 10')\n",
+    "\n",
+    "ax[1].plot(x,summed_poisson-poisson_true)\n",
+    "\n",
+    "ax[0].set_xlabel('Num Events (k)')\n",
+    "ax[0].set_ylabel('Probabilties')\n",
+    "ax[0].set_title('Overlay Plot')\n",
+    "ax[1].set_xlabel('Num Events (k)')\n",
+    "ax[1].set_ylabel('Difference')\n",
+    "ax[1].set_title('Difference in Summed Poisson Distribution with Poisson Distribution at 10')\n",
+    "\n",
+    "ax[0].legend(fontsize = 16);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b0fec222-b6ed-410b-a7dc-7ed3e15dacc0",
+   "metadata": {},
+   "source": [
+    "Here I show that after 5 days, the summed probability distribution is equivalent to a poisson distribution centered at 5x the expected value per night. The second plot shows that the difference in the two lines is nearly 0 (to a factor of 10^-15) for all values k. This makes sense conceptually, because the results of one night have no impact on the results of the next.\n",
+    "\n",
+    "### C)\n",
+    "\n",
+    "\n",
+    "When you average the summed probability distributions, the resulting distribution instead shifts to be closer to the expected value of k, and narrower as you increase the number of days. This is because as you take data over more days, the liklihood of getting outlier values over and over becomes increasingly low.\n",
```

Comment:
For part C in your explanation, the lab also asks you to discuss this conceptually in relation to part B) (how the distribution is still a Poisson), and Central Limit Theorem. Add this part and then part C will be complete.